app/controllers/cookbooks_controller.rb

```diff
@@ -281,7 +278,7 @@ def cookbook_urls_params
   end
 
   def cookbook_deprecation_params
-    params.require(:cookbook).permit(:replacement)
+    params.permit(cookbook: [:replacement])
```

Comment:
Changes the cookbook->replacement parameter from required to optionally-permitted.